pkg/cmd/dev/doctor.go

```diff
@@ -11,22 +11,64 @@
 package main
 
 import (
-	"errors"
+	"context"
+	"io/ioutil"
 	"log"
-	"os/exec"
+	"os"
+	osexec "os/exec"
 	"path/filepath"
 	"runtime"
 	"strconv"
 	"strings"
 
+	"github.com/cockroachdb/cockroach/pkg/cmd/dev/io/exec"
+	"github.com/cockroachdb/errors"
 	"github.com/spf13/cobra"
 )
 
+const devStatusFile = "bin/.dev-status"
+
+// devStatusVersion is the current "version" of the status checks performed by
+// `dev doctor``. Increasing it will force doctor to be re-run before other dev
+// commands can be run.
+const devStatusVersion = 1
```

Comment:
Nit: Can we call this `devDoctorStatusVersion` or `devSetupStatusVersion` or something similar? My worry is that `devStatusVersion` will get confused with the more general `DEV_VERSION` variable (from top-level `./dev`).

Reply:
You can if you want, you just have to make sure to update `pkg/cmd/dev/build.go` accordingly.